### PR TITLE
AwaDB new quickstart

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
# Summary

**Ticket**: https://issues.newrelic.com/browse/NR-164845
**Description**: Created AwaDB Quickstart.

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/awadb/config.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+slug: awadb
+description: |
+  ## Why should you monitor your usage of AwaDB?
+  Monitor your vector searches on AwaDB to get visibility on what you send to AwaDB, responses retrieved from AwaDB, latency, usage and errors.
+  ### Track the query performance of your Vector DB
+  Track the behavior of your vector stores. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
+  ### Track your app:  
+  By tracking key metrics like latency, throughput, error rates, and input & output, you can gain insights into your app's performance and identify areas of improvement.
+  ## Comprehensive monitoring quickstart for your AwaDB app
+  Our AwaDB quickstart provides metrics including Identify popular queries, sources, and content.
```
Comment (rahulbasu-nr) [inline quickstarts/awadb/config.yml:10]:
```suggestion
  Our AwaDB quickstart provides insights like identifying popular queries, sources, and content.
```

**Review by rahulbasu-nr — CHANGES_REQUESTED** [review]
Please make these changes

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Review by mdumpati — APPROVED** [review]
LGTM

**Pull request merged**